src/Plugin/Sampler/Bundle.php

```diff
@@ -185,11 +186,14 @@ protected function collectDefaultFieldData(FieldConfigInterface $fieldConfig, st
     $fieldType = $fieldConfig->getType();
 
     if (empty($this->collectedData[$mappedBundle]['fields'][$fieldType])) {
-      $this->collectedData[$mappedBundle]['fields'][$fieldType] = 1;
-      return;
+      $this->collectedData[$mappedBundle]['fields'][$fieldType] = [];
     }
 
-    $this->collectedData[$mappedBundle]['fields'][$fieldType]++;
+    $this->collectedData[$mappedBundle]['fields'][$fieldType][] = [
+      'is_required' => (integer) $fieldConfig->isRequired(),
```

Comment:
Maybe we should simply use `required` and `translatable` because the same property names are used in the configuration.

Now there is a question about `(integer)` vs. `(bool)`? I think bool is nicer, but I don't know is that properly parsed by YAML parser, so that you actually have bool type in the array instead of `"true"|"false"` strings.